website/content/en/configurations/identity-access-modes/_index.md

```diff
@@ -8,10 +8,12 @@ description: >
 ---
 
 ## Best Practices
+
 Following order of access modes is recommended for Secret Store CSI driver AKV provider:
 
-| Access Option 	| Comment 	|
-|---	|---	|
-| Pod Identity 	| This is the most secure way to get access to Azure resources (AKV in this case) as it uses the managed identity bound to the Pod. 	|
-| Managed Identities (System-assigned and User-assigned) 	| Managed identities eliminate the need for developers to manage credentials. Managed identities provide an identity for applications to use when connecting to Azure Keyvault. 	|
-| Service Principal 	| This is the last option to consider while connecting to AKV as access credentials need to be created as Kubernetes Secret and stored in plain text in etcd.<br>Also, this is the only option to connect to Azure resources from non Azure environment/cluster. 	|
\ No newline at end of file
+| Access Option  | Comment  |

```

Comment:
Netlify deployment is failing. Could you please fix it? It's easier to review.

Reply:
The deployment was cancelled because there was no change in docs from the previous commit in the PR. If there are no changes in between commits, it will not redeploy. The current available link is up to date: https://deploy-preview-778--secrets-store-csi-driver-provider-azure.netlify.app

![Screen Shot 2022-02-14 at 11 27 07 AM](https://user-images.githubusercontent.com/10608509/153932499-86aff216-b76f-4739-b77a-f8ccdd221bde.png)
/